appcues/src/main/java/com/appcues/ui/AppcuesOverlayViewManager.kt

```diff
@@ -70,7 +74,7 @@ internal class AppcuesOverlayViewManager(private val scope: Scope) : DefaultLife
         if (parentView.findViewById<ComposeView>(R.id.appcues_overlay_layout) == null) {
             // then we add
             val binding = AppcuesOverlayLayoutBinding.inflate(layoutInflater)
-            if (viewModel == null) viewModel = AppcuesViewModel(scope, ::onCompositionDismiss)
+            if (viewModel == null) viewModel = AppcuesViewModel(scope, renderContext, ::onCompositionDismiss)
```

Comment:
PresentingTraits needs to be aware of the RenderContext so they can create an instance of AppcuesViewModel